src/main/assets/_documentation.styl

```diff
@@ -1,4 +1,12 @@
 article
+
+  /* This takes into account the header when clicking anchor links */
+  :target:before
+    content ""
+    display block
+    height header-height
+    margin-top - header-height
+
```

Comment:
@rasummer It seems this doesn't work on Edge - would you recommend anything else?